pegen/pegen.h

```diff
@@ -44,6 +44,7 @@ typedef struct {
     int n_keyword_lists;
     void *start_rule_func;
     INPUT_MODE input_mode;
+    Py_tss_t *error_marker;
```

Comment:
If you agree we don't need to be thread-safe this could just be `jmp_buf error_marker;`.

Maybe it could be renamed `error_env`? (The `setjmp(3)` man page uses `env` as the argument name.)